tests/.pylintrc

```diff
@@ -44,7 +44,14 @@ disable=
   import-outside-toplevel,
   missing-class-docstring,
   missing-function-docstring,
-  no-self-use
+  no-self-use,
+  no-member, # because of qnode decorator
+  comparison-with-callable,
```

Comment:
Why are we ignoring this one? :)

Reply:
whenever you say `assert res.shape == x`, it thinks `res.shape` is a callable and gets angry

Reply:
Ohhh, that's tricky :) thank you :)

Reply:
Does it also complain when we use `qml.QNode` directly?

Reply:
it does not